### DISCUSSION TITLE
DynamicFilterInfo中无法动态使用 DATEDIFF

### DISCUSSION THREAD

**Discussion by sy-ing** [dotnetcore/FreeSql#752] [General]
希望能在动态条件中使用 类似于     sQueryCondi += "and DATEDIFF(MINUTE,l.Time,'" + slqp.StartTime + "') <= 0 "; 这样的方案 目前好像不支持？（我不确定）   目前只能通过动态生成语句来间接达到目的

**Comment by 2881099** [comment]
DynamicFilterInfo 不支持 SQL 操作，因为考虑到内容由前端传过来的，那等于让前端直接传 SQL 是非常危险的功能。
